File: CS/DevExpress.XtraReports.CustomControls/XRTreeListPrintCache.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraPrinting;

namespace DevExpress.XtraReports.CustomControls
{
    internal class TreeListNodePrintCache : RecordPrintCache
    {
        int nodeLevel;

        public TreeListNodePrintCache(VisualBrick brick, int nodeLevel) : base(brick)
        {
            this.nodeLevel = nodeLevel;    
        }

        public int NodeLevel { get { return nodeLevel; } }
    }
}

File: VB/DevExpress.XtraReports.CustomControls/XRTreeListPrintCache.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraPrinting

Namespace DevExpress.XtraReports.CustomControls
	Friend Class TreeListNodePrintCache
		Inherits RecordPrintCache

'INSTANT VB NOTE: The field nodeLevel was renamed since Visual Basic does not allow fields to have the same name as other class members:
		Private nodeLevel_Renamed As Integer

		Public Sub New(ByVal brick As VisualBrick, ByVal nodeLevel As Integer)
			MyBase.New(brick)
			Me.nodeLevel_Renamed = nodeLevel
		End Sub

		Public ReadOnly Property NodeLevel() As Integer
			Get
				Return nodeLevel_Renamed
			End Get
		End Property
	End Class
End Namespace
